src/style.css

```diff
@@ -1 +1,88 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
```

Comment:
Check font-styles from figma (you should use google fonts to import correct font)